.github/workflows/ci-auto.yaml

```diff
@@ -38,3 +38,8 @@ jobs:
     steps:
     - uses: actions/checkout@v3
     - run: cd nodeadm && make test-e2e
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
do we need to [make the other steps dependent](https://docs.github.com/en/actions/using-jobs/using-jobs-in-a-workflow#defining-prerequisite-jobs) on this to prevent from running potentially bad code?